src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -240,7 +226,6 @@ function ReportActionItemMessageEdit(
                 const newEmojis = EmojiUtils.getAddedEmojis(emojis, emojisPresentBefore.current);
                 if (newEmojis?.length > 0) {
                     insertedEmojis.current = [...insertedEmojis.current, ...newEmojis];
-                    debouncedUpdateFrequentlyUsedEmojis();
```

Comment:
https://github.com/Expensify/App/blob/bd1980362d1c4876858fbdc1ea5000ffa4dea2a0/src/pages/home/report/ReportActionItemMessageEdit.tsx#L239-L245

Here we can remove the above code block, as it is only setting insertedEmojis.current, and we are not using insertedEmojis.current.